airbyte-integrations/bases/standard-destination-test/src/main/resources/edge_case_messages.txt

```diff
@@ -7,3 +7,4 @@
 {"type": "RECORD", "record": {"stream": "reserved_keywords", "emitted_at": 1602637589000, "data": { "order" : "ascending" }}}
 {"type": "RECORD", "record": {"stream": "groups", "emitted_at": 1602637589000, "data": { "authorization" : "into" }}}
 {"type": "RECORD", "record": {"stream": "ProperCase", "emitted_at": 1602637589000, "data": { "ProperCase" : true }}}
+{"type": "STATE", "state": { "data": {"start_date": "2020-09-02"}}}
```

Comment:
can we add another state message to verify the last state message is output?

Reply:
yeah. good call.

Reply:
actually. we already have this in the other test case. exchange_rates_messages has multiple state messages and the test case verifies that it is the last state message that is saved.